cowprotocol/accounting/fees/classified_fees_4058574.sql

```diff
@@ -0,0 +1,60 @@
+-- This query computes fees
+--
+-- It currently uses the raw_order_rewards table which is only available on ethereum. On other chains it returns an empty table.
+--
+-- Parameters:
+--  {{start_time}} - the timestamp for which the analysis should start (inclusively)
+--  {{end_time}} - the timestamp for which the analysis should end (exclusively)
+--  {{blockchain}} - network to run the analysis on
+--
+-- The columns of the result are
```

Comment:
since the result is really on a per order basis, wouldn't it make sense to keep the uid there as well.  I know it's irrelevant for the actual slippage calculations later on but if we claim this query computes fees on a per trade basis, it is a bit strange not to include the uid